stdlib/public/core/UnicodeHelpers.swift

```diff
@@ -140,10 +140,12 @@ internal func _continuationPayload(_ x: UInt8) -> UInt32 {
   return UInt32(x & 0x3F)
 }
 
-@inlinable @inline(__always)
```

Comment:
Also this one, to double-check?

Reply:
Yeah, the inline-me fast-path has switched to the bit-check.